server/models/Document.test.js

```diff
@@ -201,7 +201,8 @@ describe("#searchForTeam", () => {
       title: "test",
     });
 
-    const results = await Document.searchForTeam(team, "test");
+    const response = await Document.searchForTeam(team, "test");
+    const { data: results } = response;
```

Comment:
Seems like we're mapping to `results` everywhere, maybe just make `data` -> `results` in the response to begin with?

Reply:
I don't mind changing it, I used data because it sounded more standard than results for an API response.